crates/y-sweet-worker/src/config.rs

```diff
@@ -7,9 +7,10 @@ const BUCKET: &str = "Y_SWEET_DATA";
 const BUCKET_KIND: &str = "BUCKET_KIND";
 const AUTH_KEY: &str = "AUTH_KEY";
 const CHECKPOINT_FREQ_SECONDS: &str = "CHECKPOINT_FREQ_SECONDS";
-const S3_ACCESS_KEY_ID: &str = "AWS_ACCESS_KEY_ID";
-const S3_SECRET_ACCESS_KEY: &str = "AWS_SECRET_ACCESS_KEY";
-const S3_REGION: &str = "AWS_REGION";
+const S3_ACCESS_KEY_ID: &str = "S3_ACCESS_KEY_ID";
```

Comment:
Let's stick to the original naming; these are canonical names used by AWS and are also used in S3-compatible systems.

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html